repokitteh.star

```diff
@@ -33,6 +33,10 @@ use(
       "label": "deps",
       "github_status_label": "any dependency change",
     },
+    {
+      "owner": "envoyproxy/tls-watchers",
+      "path": "source/extensions/transport_sockets/tls/",
```

Comment:
Should this also cover `source/common/ssl`?

Reply:
and:
```
source/extensions/common/crypto
source/extensions/filters/listener/tls_inspector
```
but we could also just add @asraa to `CODEOWNERS`.

Reply:
I wanted to avoid being a legitimate LGTM source right now, but it looks like crypto has 3 owners already, so it can't hurt to request more. 

`source/common/ssl` doesn't have a codeowner since it's core code, so i limited the watch to this